bin/cheribsdtest/cheribsdtest.c

```diff
@@ -91,7 +91,7 @@ static int fast_tests_only;
 static int qtrace;
 static int qtrace_user_mode_only;
 static int sleep_after_test;
-static int verbose;
+int verbose;
```

Comment:
Is this worth moving to a separate block?

Reply:
I can move it.  Either way is fine with me.